### PR TITLE
feature (messenger) Parse train's journey reference for miku app integration

### PR DESCRIPTION
**Pull request by qasimwarraich**
## What is this?
This allows the line number to open up the sbb miku tool in order to get more information such as the formation and exact delays. 

Additional changes:

Change the title "Liniennummer" to "Zug"
Show the type field on the mobile view (works better with new shortened column name)
Hover highlighting for new link

## Demo
demo

### CODE REVIEW

File: javascript/stationBoard-view.js [hunk 1]
```diff
@@ -88,8 +88,25 @@ let stationBoardView = {
...
-
+        let journey;
+        let journeyNumber = '';
+        if (train.journeyRef !== ''){
+            journey = train.journeyRef.split(":");
+            journeyNumber = journey.pop();
+        }
+
+        if (journeyNumber !== '') {
+            let infoUrl = "https://80e05b96-2d12-4945-b772-e22e7e1dd377-miku.app.sbb.ch/#/fahrten/direkt/" + journeyNumber;
```
Comment (fhirter) [inline javascript/stationBoard-view.js]: please put miku url into config.js

File: javascript/stationBoard-view.js [hunk 2]
```diff
@@ -88,8 +88,25 @@ let stationBoardView = {
 		let type = document.createElement("td");
 		type.classList.add("type");
 
-		type.appendChild(document.createTextNode(train.lineName));
-
+        let journey;
```
Comment (fhirter) [inline javascript/stationBoard-view.js:88]: please put new functionality into separte method.

File: styles/mobile.css [hunk 3]
```diff
@@ -3,6 +3,10 @@
         font-size: 0.7em;
     }
 
+    a:hover {
```
Comment (fhirter) [inline styles/mobile.css]: :hover makes no sense on mobile

File: styles/mobile.css [hunk 4]
```diff
@@ -18,9 +22,9 @@
         display: none;
     }
 
-    .type {
-        display: none;
-    }
+    /* .type { */
```
Comment (fhirter) [inline styles/mobile.css]: I prefere old code to be deleted instead of commented out.

**Review by fhirter — CHANGES_REQUESTED** [review]
thanks for the fine input! is the miku url confidential?

File: styles/mobile.css (hunk 4 above)
Comment (qasimwarraich) [reply styles/mobile.css]: Aha yeh me too, 4af9f4d.

File: javascript/stationBoard-view.js (hunk 1 above)
Comment (qasimwarraich) [reply javascript/stationBoard-view.js]: I agree, 5c499dc.

File: styles/mobile.css (hunk 3 above)
Comment (qasimwarraich) [reply styles/mobile.css]: Yep, curse of the copy paste. Fixed in 4af9f4d

File: javascript/stationBoard-view.js (hunk 2 above)
Comment (qasimwarraich) [reply javascript/stationBoard-view.js:88]: Added the method, c3808ab.

**Comment by qasimwarraich** [comment]
> thanks for the fine input! is the miku url confidential?

I am not 100% sure about this but it should be available for free to licensed transportation companies*.

*[Source](https://bahninfrastruktur.sbb.ch/de/produkte-dienstleistungen/bahninformatiksysteme/kundeninformation/miku.html)

**Pull request merged**